requirements.txt

```diff
@@ -10,3 +10,6 @@ torchmetrics>=0.4.1
 pyDeprecate>=0.3.1, <0.4.0
 packaging>=17.0
 typing-extensions>=4.0.0
+pytest>=7.0.1
+pytest-forked>=1.4.0
+sklearn
```

Comment:
these should not be added to the core requirements file. did you mean to add them for testing?